### PR TITLE
Fixes #28112

### LINKED ISSUE #28112: Format Typo in example
Regarding the sample code for DirectoryInfo.  The Format specifier on line 43 in the call to fi.Length.ToString("NO") I believe is a typo and should be ToString("N0") ... N Zero.

The sample program as is prints the literal word "NO" on each line.  N0 (zero) prints numeric values.

Around Line ~43.

Corrected code:  Console.WriteLine($"{fi.FullName}\t\t{fi.Length.ToString("N0")}");

https://github.com/dotnet/docs/blob/main/samples/snippets/csharp/VS_Snippets_CLR_System/system.io.directoryinfo.enumeratedirectories/cs/program.cs#L43
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 19880b28-5edd-695d-726e-4f627d00b843
* Version Independent ID: ebafb0b4-da0a-36ca-2246-df3e9d0a863e
* Content: [How to: Enumerate directories and files](https://docs.microsoft.com/en-us/dotnet/standard/io/how-to-enumerate-directories-and-files)
* Content Source: [docs/standard/io/how-to-enumerate-directories-and-files.md](https://github.com/dotnet/docs/blob/main/docs/standard/io/how-to-enumerate-directories-and-files.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @adegeo
* Microsoft Alias: **adegeo**

### PR DESCRIPTION
**Pull request by leopcode**
## Summary

Fixed typo on ToString() method calls from lines 23 an 43 in order to print numeric values.

Fixes #28112

### CODE REVIEW

**Comment by BillWagner** [comment]
Thanks for submitting this @leopcode 

This PR triggered an error in our CI build for samples. (It's historical, not related to your PR). Our samples were imported into the public repo and that operation removed the internal CI system.  

If you add a default .csproj in the folder with this change, that should fix the issue. Even better, it will help make sure we don't have errors creep into the code again later.

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=28422) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=28422) before we can accept your contribution.<br/><br/>:x: leopcode [sign now](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=28422)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/docs?pullRequest=28422) it.</sub>

**Comment by leopcode** [comment]
> Thanks for submitting this @leopcode
> 
> This PR triggered an error in our CI build for samples. (It's historical, not related to your PR). Our samples were imported into the public repo and that operation removed the internal CI system.
> 
> If you add a default .csproj in the folder with this change, that should fix the issue. Even better, it will help make sure we don't have errors creep into the code again later.

I've added a new commit. I hope I've understood you correctly.
Also let me know if you want a different name for the .csproj file.

File: samples/snippets/csharp/VS_Snippets_CLR_System/system.io.directoryinfo.enumeratedirectories/cs/enumaratedirectories.csproj [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```
Comment (BillWagner) [inline samples/snippets/csharp/VS_Snippets_CLR_System/system.io.directoryinfo.enumeratedirectories/cs/enumaratedirectories.csproj:5]: Let's go ahead and update to .NET 6. (That's been released and it's an LTS release).
```suggestion
    <TargetFramework>net6.0</TargetFramework>
```

**Review by BillWagner — COMMENTED** [review]
Thanks again @leopcode 

We have been updating our sample code to .NET 6. I made that suggestion. Once you accept it, I'll merge this PR.

Thanks again @leopcode

**Review by BillWagner — APPROVED** [review]
Thanks again @leopcode 

I'll merge this as soon as the build completes.

**Pull request merged**